src/Workspaces/CoreTestUtilities/UseExportProviderAttribute.cs

```diff
@@ -112,7 +113,14 @@ public override void After(MethodInfo methodUnderTest)
                         }
                         catch (OperationCanceledException ex) when (timeoutTokenSource.IsCancellationRequested)
                         {
-                            throw new TimeoutException("Failed to clean up listeners in a timely manner.", ex);
+                            var messageBuilder = new StringBuilder("Failed to clean up listeners in a timely manner.");
+                            foreach (var token in ((AsynchronousOperationListenerProvider)listenerProvider).GetTokens())
+                            {
+                                messageBuilder.AppendLine().Append($"  {token}");
+                            }
+
+                            Debugger.Launch();
```

Comment:
I think we normally call our fail fast helper in tests. I could imagine that if procdump is attached that counts as a debugger, which will make this call do nothing per the docs.

Reply:
Also, if I'm running from command line, I may not want a JIT debug window popping up. (If you do, congrats, you get to rewrite our helpers to do that uniformly. :smile:)

Reply:
Unintentional holdover from local debugging